src/main/java/io/atoti/spark/DatabricksManager.java

```diff
@@ -0,0 +1,188 @@
+package io.atoti.spark;
+
+import io.github.cdimascio.dotenv.Dotenv;
+import okhttp3.*;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+
+public class DatabricksManager {
+  static Dotenv dotenv = Dotenv.load();
+  static private final String clusterId = dotenv.get("clusterId");
+  static private final String token = dotenv.get("token");
+  static private final String clusterUrl = dotenv.get("clusterUrl");
+  static private final int timeOut = 10;
+  static private final int delayBeforeStateCheck = 1000;
+  static private final int periodStateCheck = 5000;
+  static private final int maxStateCheck = 60;
+  static private final MediaType JSON = MediaType.get("application/json; charset=utf-8");
```

Comment:
By convention in Java, constants use shout case. Something like:
```suggestion
  static private final String CLUSTER_ID = dotenv.get("clusterId");
  ...
```